### ISSUE TITLE
Color depth adjustement (8 or 16pbb)

### ISSUE THREAD

**Issue by TiBeN** [Plagman/gamescope#134]
Hi, 

Just tried Gamescope with some Wine games (using Lutris). This is just incredible i can now play games in full screen which only launched with a virtual desktop normally resulting a tiny window.

Some old games requires color depth to be 8 or 16 bpp. Midtown Madness is an example. The common trick to handle this is to use Xephyr which can be set to required color depth. The real downside of Xephyr is that it is not possible to get hardware acceleration (if understand well, i more of an user than a techie).

I wonder if it's possible to do the same with Gamescope (ie changing the color depth) ?

**Comment by emersion** [comment]
Which Xephyr option are you using to set the color depth to 8 or 16 bpp?

**Comment by TiBeN** [comment]
I don't use directly Xephyr but through Lutris option. But according to this the options is "-depth x" :
<https://appdb.winehq.org/objectManager.php?sClass=version&iId=4458&iTestingId=99410>

**Comment by emersion** [comment]
This doesn't seem to work, however this does:

```
Xephyr -screen 640x480x16
```

Documented here:

```
TinyX Device Dependent Usage:
-screen WIDTH[/WIDTHMM]xHEIGHT[/HEIGHTMM][+[-]XOFFSET][+[-]YOFFSET][@ROTATION][X][Y][xDEPTH/BPP[xFREQ]]  Specify screen characteristics
```

**Comment by TiBeN** [comment]
Yes you are right , i was about to edit my post as i observed the same. "-depth " is for launching a separate X server.

**Comment by emersion** [comment]
ajax said there's no way to set the depth for Xwayland right now, but that patches are welcome.

**Comment by TiBeN** [comment]
Ok. I doubt i have required skills for that but who know.. anyway thank for your feedback and investigation.

**Comment by CEbbinghaus** [comment]
Of course the 1 setting I have been trying to find / enable over the last 2 hours is no longer supported under gamescope... 

Pulling my hair out trying to launch an old game and now it turns out I cannot even utilize Xephyr 😭 

Hoping someone with the same problem has submitted a patch in the last 6 years and this issue is just out of date.
